src/order/bounded_order.lean

```diff
@@ -1737,6 +1762,61 @@ instance : complemented_lattice αᵒᵈ :=
 
 end complemented_lattice
 
+-- TODO: Define as a sublattice?
+/-- The sublattice of complemented elements. -/
+@[derive partial_order]
+def complementeds (α : Type*) [lattice α] [bounded_order α] : Type* := {a : α // is_complemented a}
+
```

Comment:
To match `is_unit.unit`:
```suggestion

def is_complemented.complemented {α : Type*} [lattice α] [bounded_order α] (a : α) (h : is_complemented a) : complementeds α := ⟨a, h⟩

```

Reply:
And this would need the appropriate `simp` lemmas back and forth with `coe` and the definitions on `complementeds`: `<`, `≤`, `\top`, `\bot`, ...

Reply:
Or do we want `complementeds.mk` to be the canonical spelling?

Reply:
I think let's keep `subtype.mk` as the canonical spelling.